incubator/hnc/pkg/reconcilers/object.go

```diff
@@ -539,6 +585,32 @@ func (r *ObjectReconciler) exclude(log logr.Logger, inst *unstructured.Unstructu
 	}
 }
 
+// addToNumPropagated adds a propagated object to the PropagatedObjects map if it does not exist.
+func (r *ObjectReconciler) addToNumPropagated(log logr.Logger, namespace, name string) {
+	r.propagatedObjectsLock.Lock()
+	defer r.propagatedObjectsLock.Unlock()
+
+	nnm := types.NamespacedName{
+		Namespace: namespace,
+		Name:      name,
+	}
+	if !r.propagatedObjects[nnm] {
+		r.propagatedObjects[nnm] = true
+	}
```

Comment:
You don't need the `if`, you can just always write.

Reply:
Done. Thanks for the suggestion!